### PR TITLE
Update Az.Storage.md

### PR DESCRIPTION
**Pull request by m2cci-LBR**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by blueww** [comment]
The description comes from https://github.com/Azure/azure-powershell/blob/main/src/Storage/Storage.Management/help/Enable-AzStorageContainerDeleteRetentionPolicy.md#L9.
Please also update this file.

**Review by isra-fel — APPROVED** [review]

File: src/Storage/Storage.Management/help/Az.Storage.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Enable last access time tracking for the Azure Storage Blob service.
 Enables Blob Restore Policy on a Storage account.
 
 ### [Enable-AzStorageContainerDeleteRetentionPolicy](Enable-AzStorageContainerDeleteRetentionPolicy.md)
-Enable delete retention policy for Azure Storage blob contatiners.
+Enable delete retention policy for Azure Storage blob containers.
```
[the remark this answers is not in the document]
Comment (isra-fel) [reply src/Storage/Storage.Management/help/Az.Storage.md:57]: Hi Wei Wei, would you open another PR to fix those typos? This one is open for too long I'm going to merge it. Thanks!

[the remark this answers is not in the document]
Comment (blueww) [reply src/Storage/Storage.Management/help/Az.Storage.md:57]: OK, merge it please

**Pull request merged**